src/components/layout/navbar.tsx

```diff
@@ -49,6 +50,7 @@ const NavBar = () => {
         <Container maxWidth="xl">
           <Toolbar disableGutters>
             <NavItems />
+            <SyncButton />
```

Comment:
This feature is only really needed for the purposes of development and synchronizing the staging timebank api, so lets have this button only render if its in staging and not production.

Reply:
The SyncButton is now rendered if the Vite DEV environment variable is true.